src/libs/DateUtils.ts

```diff
@@ -534,7 +534,7 @@ function getStatusUntilDate(inputDate: string): string {
     const endOfToday = endOfDay(now);
 
     // If the date is equal to the end of today
```

Comment:
@Krishna2323 Also should update this comment

Reply:
@Krishna2323  As the pattern of this comment, this comment gives information about when this code is executed. I am ok with your suggestion to add more explanation why using  isSameSecond

Reply:
// If the date is adjusted to the following day, use isSameSecond to ensure accuracy to millisecond

Reply:
Ok. It looks good

// If the date is adjusted to the following day
// Use isSameSecond to ensure accuracy, otherwise, setting the time to 11:59PM would also display until tomorrow.

Reply:
@Krishna2323 It is a bit lengthy. I think this is enough 

// If the date is adjusted to the following day